From 096973e328377bcf133e7b495581febc42cb6fb0 Mon Sep 17 00:00:00 2001
From: "bo.yang" <bo.yang@amlogic.com>
Date: Tue, 27 May 2014 19:08:11 +0800
Subject: [PATCH 4223/5965] vfm: support vfm to be configurable

this commit is usable while bringuping when video modules are not needed yet.
---
 drivers/amlogic/Kconfig      | 1 +
 drivers/amlogic/Makefile     | 2 +-
 drivers/amlogic/vfm/Kconfig  | 9 +++++++++
 drivers/amlogic/vfm/Makefile | 2 +-
 4 files changed, 12 insertions(+), 2 deletions(-)
 create mode 100755 drivers/amlogic/vfm/Kconfig
 mode change 100644 => 100755 drivers/amlogic/vfm/Makefile

diff --git a/drivers/amlogic/Kconfig b/drivers/amlogic/Kconfig
index 814441d6d0a0..d30a61489ea1 100755
--- a/drivers/amlogic/Kconfig
+++ b/drivers/amlogic/Kconfig
@@ -67,6 +67,7 @@ source "drivers/amlogic/audiodsp/Kconfig"
 #  Video
 #
 comment "Video devices"
+source "drivers/amlogic/vfm/Kconfig"
 source "drivers/amlogic/amports/Kconfig"
 source "drivers/amlogic/canvas/Kconfig"
 source "drivers/amlogic/display/Kconfig"
diff --git a/drivers/amlogic/Makefile b/drivers/amlogic/Makefile
index 30a4b67fd795..966555fe4da4 100755
--- a/drivers/amlogic/Makefile
+++ b/drivers/amlogic/Makefile
@@ -59,7 +59,7 @@ obj-$(CONFIG_AMLOGIC_SPI_NOR) += spi_nor/
 #obj-$(CONFIG_CARDREADER) += cardreader/
 obj-$(CONFIG_MMC_AML) +=mmc/
 
-obj-y += vfm/
+obj-$(CONFIG_AML_VFM) += vfm/
 
 obj-$(CONFIG_AML_AUDIO_DSP) += audiodsp/
 
diff --git a/drivers/amlogic/vfm/Kconfig b/drivers/amlogic/vfm/Kconfig
new file mode 100755
index 000000000000..1ba7486513b2
--- /dev/null
+++ b/drivers/amlogic/vfm/Kconfig
@@ -0,0 +1,9 @@
+#
+# VFM configuration
+#
+
+config AML_VFM
+	bool "Amlogic VFM Support"
+	default y
+	help
+	  Select to enable VFM support
diff --git a/drivers/amlogic/vfm/Makefile b/drivers/amlogic/vfm/Makefile
old mode 100644
new mode 100755
index 09512362d96f..855404c789af
--- a/drivers/amlogic/vfm/Makefile
+++ b/drivers/amlogic/vfm/Makefile
@@ -2,7 +2,7 @@
 # Makefile for VFM
 #
 
-obj-y += aml_vfm.o
+obj-$(CONFIG_AML_VFM) += aml_vfm.o
 aml_vfm-objs := vfm.o vframe_provider.o vframe_receiver.o
 
 
-- 
2.19.0

